README.md.old

```diff
@@ -1,4 +1,18 @@
-{{ ossHeader }}
+<head>
+  <meta name="Momento Node.js Client Library Documentation" content="Node.js client software development kit for Momento Serverless Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md) 
+
+# Momento Node.js Client Library
+
+
+Node.js client SDK for Momento Serverless Cache: a fast, simple, pay-as-you-go caching solution without
```

Comment:
Here's another instance where we need to drop "Serverless" from the product name